src/pages/CartPage.jsx

```diff
@@ -0,0 +1,21 @@
+import { Flex } from "@sparrowengg/twigs-react";
+import SideBar from "../components/SideBar";
+import CartItems from "../components/CartItems";
+import BagItems from "../components/BagItems";
+import { useSelector } from "react-redux";
+const CartPage = () => {
+  const cart = useSelector((state) => state.cart);
+  return (
+    <>
```

Comment:
@arun-121 can you remove unused react fragements

Reply:
removed unused fragments